modules/monitoring/application/controllers/ServicesController.php

```diff
@@ -153,6 +155,16 @@ public function showAction()
         $this->view->sendCustomNotificationLink = Url::fromRequest()->setPath(
             'monitoring/services/send-custom-notification'
         );
+
+        $this->view->extensionsHtml = array();
+        foreach (Hook::all('Monitoring\DetailviewExtension') as $hook) {
+            /** @var DetailviewExtensionHook $hook */
+            $module = $this->view->escape($hook->getModule()->getName());
+            $this->view->extensionsHtml[] =
```

Comment:
The extension should only be added to the array if the hook does not return the empty string. Else we may render empty containers.